src/util/config/config.cpp

```diff
@@ -698,10 +698,6 @@ namespace dxvk {
     { R"(\\SWTFU2\.exe$)", {{
       { "d3d9.forceSamplerTypeSpecConstants",  "True" },
     }} },
-    /* Scrapland (Remastered)                   */
-    { R"(\\Scrap\.exe$)", {{
-      { "d3d9.deferSurfaceCreation",        "True" },
```

Comment:
Why is this moved to d3d8? Is it a d3d8 game?

Reply:
Yes. It was in d3d9 for people using d3d8to9